src/blenderbim/blenderbim/bim/module/pset/prop.py

```diff
@@ -173,3 +173,26 @@ class WorkSchedulePsetProperties(PropertyGroup):
     active_pset_name: StringProperty(name="Pset Name")
     properties: CollectionProperty(name="Properties", type=Attribute)
     pset_name: EnumProperty(items=getWorkSchedulePsetNames, name="Pset Name")
+    
+
+class PropertiesToRename(PropertyGroup):
+    pset_name: StringProperty(name="Pset")
+    existing_property_name: StringProperty(name="Existing Property Name")
+    new_property_name: StringProperty(name="New Property Name")
+    
+    
+class PropertiesToAddOrEdit(PropertyGroup):
+    pset_name: StringProperty(name="Pset")
+    property_name: StringProperty(name="Property")
```

Comment:
Maybe it would be simpler to re-use the [Attribute](https://github.com/IfcOpenShell/IfcOpenShell/blob/3accc1dbd6510d176455e8f163fdbfe157762bdc/src/blenderbim/blenderbim/bim/prop.py#L160-L213) property group ?

Reply:
Sure - i don't mind trying that.